performance_manager/startup.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import logging
+import os
+import pathlib
+import sched
+import sys
+import time
+
+from typing import Dict, List
+
+import sqlalchemy as sa
+from sqlalchemy.orm import sessionmaker
+
+from lib import (
+    SqlBase,
+    MetadataLog,
+    get_experimental_engine,
+    get_local_engine,
+    process_vehicle_positions,
+)
+
+logging.getLogger().setLevel("INFO")
+
+DESCRIPTION = """Entry Point to RDS Manipulating Performance Manager Scripts"""
+HERE = os.path.dirname(os.path.abspath(__file__))
+
+
+def load_environment() -> None:
+    """
+    boostrap .env file for local development
+    """
+    try:
+        if int(os.environ.get("BOOTSTRAPPED", 0)) == 1:
+            return
+
+        env_file = os.path.join(HERE, "..", ".env")
+        logging.info("bootstrapping with env file %s", env_file)
+
+        with open(env_file, "r", encoding="utf8") as reader:
+            for line in reader.readlines():
+                line = line.rstrip("\n")
+                line.replace('"', "")
+                if line.startswith("#") or line == "":
+                    continue
+                key, value = line.split("=")
+                logging.info("setting %s to %s", key, value)
+                os.environ[key] = value
+
+    except Exception as exception:
+        logging.error("error while trying to bootstrap")
+        logging.exception(exception)
+        raise exception
+
+
+def parse_args(args: list[str]) -> argparse.Namespace:
```

Comment:
same parse args plus an interval argument for how frequently to run the event loop.